distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.security.response.filter.ResponseHeaderConfig.adoc

```diff
@@ -0,0 +1,34 @@
+.[[org.codice.ddf.security.response.filter.ResponseHeaderConfig]]HTTP Response Security Configuration
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Content Security Policy
+|xContentSecurityPolicy
```

Comment:
The actual variable name for the _Content Security Policy_ is `xContentSecurityPolicy`